executor/executor_test.go

```diff
@@ -456,7 +456,7 @@ func (s *testSuite) TestAdminShowDDLJobs(c *C) {
 	err = tk.Se.CommitTxn(context.Background())
 	c.Assert(err, IsNil)
 
-	re = tk.MustQuery("admin show ddl jobs 1")
```

Comment:
Prefer to add some case to `admin show ddl jobs 1 where ...` instead of deleting the former case.